### ISSUE TITLE
Connection Pool settings

### ISSUE THREAD

**Issue by respergu** [codecentric/gatling-jdbc#9]
Currently we are using default connection pool configuration, 

ConnectionPool.singleton(url, username, pwd)

it would be nice to be able to configure the scalikejdbc connection pool
Seems that in some cases max size 8 is not enough when doing load testing.

**Comment by rbraeunlich** [comment]
Hi @respergu
I agree, the connection pool should be configurable. I'll take a look into that.

**Comment by rbraeunlich** [comment]
Hi @respergu 
I released jdbc-gatling 2.1.0. There you can configure your connection pool like this:
```
    val settings = ConnectionPoolSettings(maxSize = 20)
    jdbc
      .url("jdbc:h2:mem:test;DB_CLOSE_ON_EXIT=FALSE")
      .username("sa")
      .password("sa")
      .driver("org.h2.Driver")
      .connectionPoolSettings(settings)
      .build
```

**Comment by respergu** [comment]
Thanks! this is really useful!
